.circleci/config.yml

```diff
@@ -0,0 +1,88 @@
+version: 2
+jobs:
+  install:
+    macos:
+      xcode: "9.2.0"
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - restore_cache:
+          key: cache-{{ checksum "yarn.lock" }}
+      - run:
+          name: Installing Dependencies
+          command: yarn --ignore-engines
+      - save_cache:
+          key: cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run:
+          name: Getting build icon
+          command: if [[ $CIRCLE_BRANCH == canary ]]; then cp build/canary.icns build/icon.icns; fi
+      - persist_to_workspace:
+          root: .
+          paths:
+            - node_modules
+
+  test:
+    macos:
+      xcode: "9.2.0"
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Testing
+          command: yarn test
+
+  build:
+    macos:
+      xcode: "9.2.0"
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Building
+          command: yarn dist --publish 'never'
+      - store_artifacts:
+          path: dist
+      - persist_to_workspace:
+          root: .
+          paths:
+            - dist
+
+  release:
+    macos:
+      xcode: "9.2.0"
+    steps:
+      - checkout
+      - run:
+          name: Deploying to GitHub
+          command: yarn dist
+
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - install
+      - build:
+          requires:
+            - install
+          filters:
+            branches:
+              only:
+                - master
+                - canary
+                - circleci-2
```

Comment:
Should this be here in this PR or was it for testing 🕵️

Reply:
Good spot, I just didn't save the file 😅